### ISSUE TITLE
Impossible to add new converters

### ISSUE THREAD

**Issue by brotherseb** [heiglandreas/htmltopdflib#5]
Hello !

I tried to add a new converter but it was not used, because (i think) there is a missing line in your code :) 

In class Factory, line 51 :
self::$factory = $this; 

Without this set, Factory::factory will return self::withDefaultConverters() (and then reset the converters list).

**Comment by heiglandreas** [comment]
Have you tried the method described in the README for full flexibility?

```php
$converter = new Converter(Factory::fromConverterList(ConverterList::createViaReflection([
    'em' => Em::class,
    'li' => Li::class,
    'ol' => Ol::class,
    'p' => P::class,
    'strong' => Strong::class,
    'ul' => Ul::class,
    'myTag' => MyConverterClass::class,
])));
echo $converter->convert($htmlContent);
```

The `Factory::factory()` method is on purpose using only the default converters. That is to make it as easy as possible for those that do not want to extend anything.

**Comment by brotherseb** [comment]
Yes, unfortunately :(
It's quite easy to reproduce, i added a "h1" tag with a class i created (copied / pasted P.php and replaced macro value with "h1"). 

From what i understand, the problem is when Standard.getContent is called. This line :
return Factory::factory($node, $this)->getPdfLibString();

calls the factory method, which creates a new factory from default.

If it can help...

**Comment by heiglandreas** [comment]
This should be fixed in the new Version 2.0.1.

Thanks for bringing that to my attention! I'm not 100% happy yet with the fix but it works for now :wink:

The issue was indeed the Standard-converter that was used whenever an unknown tag was encountered and then overwrote the ConverterList with the default values. Sadly that happens on encountering the first tag which is always `html` :frowning_face:

**Comment by brotherseb** [comment]
Thanks a lot for your fix!
It works now :)
